cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/vsphere/vsphere-mc-common/VsphereCredentialsStep.tsx

```diff
@@ -1,18 +1,33 @@
 // React imports
-import React, { useState } from 'react';
+import React, { ChangeEvent, useContext, useState } from 'react';
 import { SubmitHandler, useForm } from 'react-hook-form';
 // Library imports
 import { CdsButton } from '@cds/react/button';
 import { CdsIcon } from '@cds/react/icon';
 import { yupResolver } from '@hookform/resolvers/yup/dist/yup';
 // App imports
-import { FormInputs } from '../../aws/wizard-basic/management-credential-step/ManagementCredentials';
 import { StepProps } from '../../../../shared/components/wizard/Wizard';
 import { vsphereCredentialFormSchema } from './vsphere.credential.form.schema';
+import { CdsInput } from '@cds/react/input';
```

Comment:
small nit, pls move lines 11-12 library imports

Reply:
will do in next checkin